Makefile

```diff
@@ -1,22 +1,47 @@
-PKGS := $(shell go list ./... | grep -v /vendor)
+VENDOR := vendor
+PKGS := $(shell go list ./... | grep -v /$(VENDOR)/)
+SRC = $(shell find . -type f -name '*.go' -not -path "./$(VENDOR)/*")
 BIN_DIR := $(GOPATH)/bin
-DEP_BIN := $(BIN_DIR)/dep
+DEP := $(BIN_DIR)/dep
 GOMETALINTER := $(BIN_DIR)/gometalinter
 
-dep-ensure: $(DEP_BIN)
-	$(DEP_BIN) ensure -v
-
-$(DEP_BIN):
-	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
 .PHONY: test
-test: dep-ensure
+test: dep fmtcheck vet
 	go test $(PKGS)
 
+$(DEP):
+	go get -u github.com/golang/dep/cmd/dep
+
+.PHONY: dep
+dep: $(DEP)
+	$(DEP) ensure
+	(cd ghwc/ && $(DEP) ensure)
```

Comment:
oops :) need to remove this one line

Reply:
Yeah, saw that in travis.  Fix incoming.